checker.js

```diff
@@ -1,16 +1,31 @@
 const expect = require('chai').expect;
+var checkall=require("./Conditions.js").checkall;
 
 function vitalsAreOk(bpm, spo2, respRate) {
-    if(bpm < 70 || bpm > 150) {
-        return false;
-    } else if(spo2 < 90) {
-        return false;
-    } else if(respRate < 30 || respRate > 95) {
-        return false;
-    }
-    return true;
+  return checkall(bpm,spo2,respRate);
+    
 }
 
+let Bp_limits = [70, 150];
+let spo2_limits = [30 , 100];
+let respRate_limits = [30 , 95]
+
+function isBpOk(bpm)
+{
+    return isVital_LimitsOk(bpm, Bp_limits[0], Bp_limits[1]);
+}
+function isSpo2Ok(spo2)
+{
+    return isVital_LimitsOk(spo2, spo2_limits[0], spo2_limits[1])
+}
+function isRespRateOk(respRate)
+{
+    return isVital_LimitsOk(respRate, respRate_limits[0], respRate_limits[1])
+}
+function isVital_LimitsOk(val , low, high){
```

Comment:
The abstraction of range-check is good, but looks like it is never called?
vitalsAreOk calls checkall, which does not use the range-check isVital_LimitsOk function.

Reply:
Yes, the vitalsAreOk make an overall check by confirming that spoc2, resprate and bp are all within limits. That functionality is mentioned in checkall() which i have imported. The isVitalsLimitsOk checks if individual features are alright